packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -480,6 +504,57 @@ QUnit.module('interaction via swipe', {
         });
     });
 
+    QUnit.test('when only one item is visible, swipe action does not move the current visible item', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false }
+            ]
+        });
+        const instance = $multiView.dxMultiView('instance');
+        const $itemContainer = $multiView.find(toSelector(MULTIVIEW_ITEM_CONTAINER_CLASS));
+        const pointer = pointerMock($multiView);
+
+        pointer.start().swipeStart().swipe(0.1).swipeEnd(1);
+        assert.strictEqual(position($itemContainer), 0, 'container did not move');
+        assert.strictEqual(instance.option('selectedIndex'), 0, 'selectedIndex is not changed');
+    });
+
+    QUnit.test('swiping right should skip hidden items', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false },
+                { text: '3', visible: true }
+            ]
+        });
+        const instance = $multiView.dxMultiView('instance');
+        const $itemContainer = $multiView.find(toSelector(MULTIVIEW_ITEM_CONTAINER_CLASS));
+        const pointer = pointerMock($multiView);
+
+        pointer.start().swipeStart().swipe(0.1).swipeEnd(1);
+        assert.strictEqual(position($itemContainer), $itemContainer.width() / 10, 'container did move');
+        assert.strictEqual(instance.option('selectedIndex'), 0, 'selectedIndex is not changed');
```

Comment:
[incorrect assrtion]
But it should be changed to show third item..
```suggestion
        assert.strictEqual(instance.option('selectedIndex'), 2, 'selectedIndex is not changed');
```

Reply:
[2b58172](https://github.com/DevExpress/DevExtreme/pull/28128/commits/2b5817238d52c2df9b3704733c5f384cedc42c1c)